hypothesis-python/src/hypothesis/extra/pytestplugin.py

```diff
@@ -19,14 +19,17 @@
 
 import pytest
 
-from hypothesis import core, settings
+from hypothesis import Verbosity, core, settings
 from hypothesis.reporting import default as default_reporter
 from hypothesis.reporting import with_reporter
 from hypothesis.statistics import collector
 from hypothesis.internal.compat import OrderedDict, text_type
 from hypothesis.internal.detection import is_hypothesis_test
 
 LOAD_PROFILE_OPTION = '--hypothesis-profile'
+VERBOSITY_OPTION = '--hypothesis-verbosity'
+# subsituted with the profile name and the verbosity level.
+VERBOSITY_PROFILE_NAME = '%s-with-%s-verbosity'
```

Comment:
I would delete this constant and simply use the string inline - the global constants are to ensure that functions are consistent, but the profile name is only ever used in one place.